desktop/dev-environments.md

```diff
@@ -43,6 +43,10 @@ If it doesn't detect Git as a valid command, you must reinstall Git and ensure y
 
 ![Windows add Git to path](images/dev-env-gitbash.png){:width="300px"}
 
+> **Note**
+>
+> You might need to quit and restart entirely Docker Desktop once git has been installed. From the Docker menu, select **Quit Docker Desktop**, then restart it.
```

Comment:
I don't think we need the `might` here, people must restart Docker Desktop to make the detection work properly, no? 

```suggestion
> You need to quit and restart entirely Docker Desktop once git has been installed. From the Docker menu, select **Quit Docker Desktop**, then restart it.
```